cores/esp32/esp32-hal-uart.h

```diff
@@ -56,15 +56,16 @@ void uartEnd(uart_t* uart);
 
 uint32_t uartAvailable(uart_t* uart);
 uint32_t uartAvailableForWrite(uart_t* uart);
-uint8_t uartRead(uart_t* uart);
-uint8_t uartPeek(uart_t* uart);
+int16_t uartRead(uart_t* uart);
```

Comment:
better use `int` if size of the value is not important. It was 8bit by default, because uart is 8 bit.